ab_testing_api_sample_scripts/create_ab_test.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+
+
+while getopts t:p:g:1:2: flag
+do
+    case "${flag}" in
+        t) TOKEN=${OPTARG};;
+        p) PAGE_ID=${OPTARG};;
+        g) OPTIMIZATION_GOAL=${OPTARG};;
+        1) VIDEO_ID_1=${OPTARG};;
+        2) VIDEO_ID_2=${OPTARG};;
+    esac
+done
+
+usage () {
+    echo "Usage: create_ab_test.sh -t <api_token> -p <page_id> -g <optimization_goal> -v1 <video_id_1> -v2 <video_id_1>";
```

Comment:
`-p` or `-o` for the page ID? let's be consistent between the two scripts. Also, shouldn't this be `-1` and `-2`? instead of `-v1` and `-v2`? Otherwise, you get this warning:
```
./create_ab_test.sh: illegal option -- v
./create_ab_test.sh: illegal option -- v
```